buddyboss/buddyboss-platform
# 1566 - Fixed the issue with content, subject and link when Wishlist Member plugin is activated

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1566.

### How to test the changes in this Pull Request:

1. Go to '.Admin Dashboard' and activate the WishList Member Plugin.
2. Now go to profile account >> export data
3. Request to Export Data.
4. You will receive an Email Confirmation.
5. Click on the confirmation link and See error.

### Proof Screenshots or Video
https://www.loom.com/share/b375f695db074b63b6258812ca8776b5?from_recorder=1



### Other information:
